### PR TITLE
Fix Ask deadlock #2

### PR DESCRIPTION
**Pull request by 0x53A**
https://github.com/akkadotnet/akka.net/pull/2548

### CODE REVIEW

File: src/core/Akka/Actor/Futures.cs [hunk 1]
```diff
@@ -128,10 +128,18 @@ internal static IActorRefProvider ResolveProvider(ICanTell self)
             return null;
         }
 
+        private const int RunContinuationsAsynchronously = 64;
+        private static bool isRunContinuationsAsynchronouslyAvailable = Enum.IsDefined(typeof(TaskCreationOptions), RunContinuationsAsynchronously);
```
Comment (Aaronontheweb) [inline src/core/Akka/Actor/Futures.cs]: @0x53A mark as `readonly`

**Comment by Aaronontheweb** [comment]
@0x53A missing a public API approval: http://getakka.net/community/public-api-changes.html

**Comment by Aaronontheweb** [comment]
@0x53A and the other PR we re-opened, should still look at merging that in addition to this one, correct?

**Comment by 0x53A** [comment]
This one supersedes the other one - you can decide which one to take (this branch is even based on the other one, but I would rebase it before you merge)

**Pull request merged**